python/build-wheel-manylinux2014.sh

```diff
@@ -54,6 +60,8 @@ for ((i=0; i<${#PYTHONS[@]}; ++i)); do
   # dashboard directory.
   git clean -f -f -x -d -e .whl -e python/ray/new_dashboard/client -e dashboard/client
 
+  export BAZEL_LINKLIBS="-l%:libstdc++.a"
```

Comment:
Hmm, does this mean we are linking `libstdc++` statically? That looks a little sketchy to me. Probably we should dynamically link against the C++ standard library that comes with the manylinux2014 container.

Reply:
Yep. That's exactly what's happening though -- we're linking the libstdc++ in the manylinux2014 container.

Reply:
I think this is linking it statically, but we should be linking it dynamically. I have seen problems in the past where we have had two different versions of the standard library used in one executable which produced segfaults because some global objects crashed. My understanding is that this could be happening with this if e.g. a python module imports the c++ standard lib dynamically.

Reply:
@barakmich What happens if you remove this line, does it still work?